src/nanovg.c

```diff
@@ -2119,22 +2119,30 @@ void nvgRect(NVGcontext* ctx, float x, float y, float w, float h)
 
 void nvgRoundedRect(NVGcontext* ctx, float x, float y, float w, float h, float r)
 {
-	if (r < 0.1f) {
-		nvgRect(ctx, x,y,w,h);
+	nvgRoundedRectEx(ctx, x, y, w, h, r, r, r, r);
+}
+
+void nvgRoundedRectEx(NVGcontext* ctx, float x, float y, float w, float h, float r1, float r2, float r3, float r4)
```

Comment:
I agree with the name, the Ex kinda brought back bad win32api dreams.

The ordering of the radii parameters was because of the order they were being drawn, in my actual C++ wrapper it's just the way you suggested. It definitely makes sense.